src/AutoUI/index.tsx

```diff
@@ -721,7 +736,12 @@ const getColumnsFromSchema = <T extends AutoUIBaseResource<T>>({
 				priority,
 				type: 'predefined',
 				refScheme: refScheme?.[0],
-				sortable: customSort?.[key] ?? getSortingFunction(key, val),
+				sortable:
+					(xNoSort ? false : null) ??
+					(typeof customSort?.[key] !== 'string'
+						? (customSort?.[key] as TableSortFunction<T>)
+						: null) ??
+					getSortingFunction(key, val),
```

Comment:
Should also check to make sure nobody is passing functions for server side pagination mode or strings for in-memory mode

Reply:
👍